### PR TITLE
Add whatsnew page for Firefox 73 (Fixes #8436)

### LINKED ISSUE #8436: WNP73: Code
## Description

In #8428 we'll build several campaign pages. This card is about coding WNP73, which will be a primary entry point for the campaign. 

* WNP73 will take the copy from the campaign landing page
* It will have 3 content items below the main CTA area; [WNP70 experiment v5](﻿﻿https://docs.google.com/presentation/d/1PlWlpWK3L27W2zmZ2Jp1VVhbDwUGMPohHCOEIQRl-wM/edit#slide=id.g7c4f53cf2e_0_1222) is an approach that could work. 
* it will have a state-based CTA with two states. IF Firefox is the default, the CTA will be the mobile send-to-device widget. If Firefox is not the default, the CTA will encourage users to set Firefox as the default browser. The set-default CTA will link to the campaign's thanks page.

Designs for the campaign landing page (https://github.com/mozilla/bedrock/issues/8429) will be very helpful in building the WNP73. 

[Copy doc for WNP73](https://docs.google.com/document/d/1LzZfDKW-UK-QcD3qaY59jXiwiNZTmr2adcr2-2VejV0/edit#)

[Releng bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1609993)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] WNP73 is launched in production by 2020/01/23
- [ ] All approvals complete
- [ ] Ready for staff localization in Tier 1
- [ ] Strings extracted for community localization

#### Approvals
- [ ] RASCI A
- [ ] PM

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description

[Copy doc for WNP73](https://docs.google.com/document/d/1LzZfDKW-UK-QcD3qaY59jXiwiNZTmr2adcr2-2VejV0/edit#)

[Design](https://github.com/mozilla/bedrock/issues/8442)

TODO:
- [ ] tests.
- [ ] detect if Firefox is preferred browser JS.

## Issue / Bugzilla link
#8436

## Testing
Demo URL: https://www-demo5.allizom.org/en-US/firefox/73.0/whatsnew/all/

### CODE REVIEW

File: tests/functional/firefox/whatsnew/test_whatsnew_73.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import pytest
+
+from selenium.common.exceptions import TimeoutException
+from pages.firefox.whatsnew.whatsnew_73 import FirefoxWhatsNew73Page
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_get_mobile_modal_displayed(base_url, selenium):
+    page = FirefoxWhatsNew73Page(selenium, base_url, locale='sv-SE').open()
```
Comment (alexgibson) [inline tests/functional/firefox/whatsnew/test_whatsnew_73.py]: This test will fail in production unless the `sv-SE` locale is activated, just a heads up

Comment (amychurchwell) [reply tests/functional/firefox/whatsnew/test_whatsnew_73.py]: Would there be a better way to write this test? Unsure how to write a condition for "has firefox set as default already" specifically...

Comment (alexgibson) [reply tests/functional/firefox/whatsnew/test_whatsnew_73.py]: Keep in mind we can't do automated tests for things that rely on UItour, or browser features that depend on system settings. If this CTA is shown to Firefox users only if it's default then that may cause issues in automation.

Also, please make sure to push your tests to run-integration-tests branch and make sure they pass.

Comment (amychurchwell) [reply tests/functional/firefox/whatsnew/test_whatsnew_73.py]: All of these tests are conditional on uitour, I'm thinking I should remove the tests altogether?

Comment (alexgibson) [reply tests/functional/firefox/whatsnew/test_whatsnew_73.py]: You can test whatever the default state for the page is.

**Comment by alexgibson** [comment]
Moving this to a branch on the mozilla/bedrock repo. Closing in favor of https://github.com/mozilla/bedrock/pull/8458

**Pull request closed without merging**